Directory.Build.props

```diff
@@ -11,11 +11,11 @@
     <LangVersion>10.0</LangVersion>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(DisableArcade)' == '1'">
```

Comment:
Why is this being removed?

Reply:
Because it had no impact. The property being set was identical between the two conditions. If you want it back, can you suggest how it should be defined given these current changes?